gce/image/scylla_gce.json

```diff
@@ -19,6 +19,8 @@
       "omit_external_ip": false,
       "disk_size": 20,
       "image_labels":  {
+          "branch": "{{`user branch`| clean_resource_name}}",
```

Comment:
This should probably be named `scylla_branch`.

Reply:
It is making sense, but no.
Yaron set this on AMI, to differentiate between the "production" AMIs and our testing AMIs, so we have "master" or "branch-4.5" for the production builds and on the other hand pkg-branch such as "fix-gce-label" as an ID when we debug.
I agree that the name is misleading.
What a better name should be (both here, and for AMI tag, depends on QA - as need to change their query I guess)?

Reply:
Since the QA are looking for scylla_branch, when ever we did debug sessions it was always pointing to scylla's stable branch. causing them to use our debug AMI instead of the offical one. 

So it's actually scylla-pkg branch

Reply:
What is the purpose of this information? Showing scylla.pkg git branch name does not seem like something we want our end user to see...

Reply:
These "internal" testing items should not be visible to end users. Are they? IF so - we should put these somewhere else.
I also think that "master" items should not be visible to end users... before we run any test on them. We can set them as visible after the test passes (as we have for other install ways - the "latest nightly unstable".